pulsar-io/debezium/oracle/pom.xml

```diff
@@ -39,16 +39,49 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.debezium</groupId>
+      <artifactId>debezium-connector-oracle</artifactId>
+      <version>${debezium.oracle.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.debezium</groupId>
+      <artifactId>debezium-core</artifactId>
+      <version>${debezium.oracle.version}</version>
+    </dependency>
```

Comment:
I don't think it's a great idea to have a separate version of debezium-core when Oracle connector is used. This will cause class version conflicts as we have seen before. Please see #22668 as an example of this.

I think that should be upgrading Debezium in all connectors at the same time to avoid class version conflicts and the confusion caused for maintenance when multiple versions of Debezium is used.

Reply:
+1